include/libswiftnav/baseline.h

```diff
@@ -25,13 +25,20 @@ void amb_from_baseline(u8 num_dds, const double *DE, const double *dd_obs,
 s8 lesq_solution_float(u8 num_dds, const double *dd_obs, const double *N,
                        const double *DE, double b[3], double *resid)
                          __attribute__((warn_unused_result));
+
 s8 lesq_solution_int(u8 num_dds, const double *dd_obs, const s32 *N,
-                     const double *DE, double b[3], double *resid)
+                     const double *DE, double b[3])
                        __attribute__((warn_unused_result));
 
-void least_squares_solve_b_external_ambs(u8 num_dds, const double *ambs,
+s8 least_squares_solve_b_external_ambs(u8 num_dds, const double *ambs,
          const sdiff_t *sdiffs_with_ref_first, const double *dd_measurements,
          const double ref_ecef[3], double b[3]);
```

Comment:
Or perhaps hold off until we merge my PR as this will prevent building against a stock dgnss_management.c